components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/service/DeviceManagementProviderServiceImpl.java

```diff
@@ -160,6 +160,15 @@ public FeatureManager getFeatureManager(String deviceType) {
 
     @Override
     public boolean enrollDevice(Device device) throws DeviceManagementException {
+        if (device != null) {
+            if (log.isDebugEnabled()) {
+                log.debug("Enrolling the device " + device.getId() + "of type '" + device.getType() + "'");
+            }
+        } else {
+            String msg = "required values are not set for device enrollment";
```

Comment:
Use proper capitalization in error messages. Please do the same in similar places.